dcrtweetbot.go

```diff
@@ -24,9 +24,15 @@ var shell *ipfs.Shell
 var client *twitter.Client
 var config *Config
 var dcrtimeHost string
-var timestampedTweets = make([]tweetResult, 100)
+var timestampedTweets = make([]displayTweet, 100)
 var count int
 
+type displayTweet struct {
```

Comment:
This struct seems unnecessary to me. We can just use `tweetResult` when displaying tweets. Also, we timestamp the tweet thread entirely so in our UI we will need to display that.
By the entire thread, I mean: target tweet + parent tweets.

Reply:
I needed to create this struct basically to flatten the `tweetResult.Tweet` struct in order to use nested fields from the `Tweet` struct [here](https://github.com/fernandoabolafio/dcrtweetbot/pull/24/files#diff-8e64dd1abedc6f31a3430d556c8abe41R5).

Using `$tweet.Tweet.text` in `public/templates/tweet.html` didn't work. is there a way to access nested struct fields in the template directly ?